consensus/polybft/consensus_runtime.go

```diff
@@ -206,7 +219,7 @@ func (c *consensusRuntime) initCheckpointManager(logger hcf.Logger) error {
 		c.checkpointManager = newCheckpointManager(
 			wallet.NewEcdsaSigner(c.config.Key),
 			defaultCheckpointsOffset,
```

Comment:
Shouldn't you retrieve this value from `NetworkParams`?

Reply:
Nice catch :100: .
Fix: https://github.com/0xPolygon/polygon-edge/pull/1519/commits/b0b127e910c07f81d536feec00f6c15005c0eee6